### PR TITLE
1.0.3 notes

### PR DESCRIPTION
**Pull request by mattcary**
/kind documentation

```release-note
None
```

/assign @saikat-royc

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.0.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# v1.0.3 - Changelog since v1.0.2
+
+### Other
+
+- Fix build for various go versions.
```
Comment (saikat-royc) [inline CHANGELOG/CHANGELOG-1.0.md:5]: PR link

Comment (mattcary) [reply CHANGELOG/CHANGELOG-1.0.md]: done

**Comment by mattcary** [comment]
integration build succeeded, force-merging due to known failure.

**Pull request merged**